src/chains/ethereum/ethereum/src/helpers/gas-estimator.ts

```diff
@@ -34,7 +62,10 @@ const stepTracker = () => {
         preCompileCheck = false;
       }
       if (isCall(info.opcode.name)) {
-        info.stack = info.stack.map(val => val.clone());
+        // this might not be necessary. previously this was a BN that we were
+        // cloning, I'm guessing to get the values and not have refs. I didn't
+        // know of a good way to do this with bigints, or if it's needed at all
+        info.stack = info.stack.map(val => Quantity.toBigInt(val));
```

Comment:
You shouldn't have to clone the bigints themselves as they're immutable primitives. You might need to clone the array though, something like:

```suggestion
        info.stack = [...info.stack];
```